spec/system/task_spec.rb

```diff
@@ -1,13 +1,26 @@
 require 'rails_helper'
 RSpec.describe 'タスク管理機能', type: :system do
-
+  before do
+    task1 = FactoryBot.create(:task)
+    task2 = FactoryBot.create(:second_task)
+    task3 = FactoryBot.create(:third_task)
```

Comment:
`config.include FactoryBot::Syntax::Methods`を設定してるので、
`FactoryBot.create(:task)`は省略して記載できます。
```suggestion
    task3 = create(:third_task)
```

Reply:
こちらも書き換えました。